python/cudaq/_cudaq.cpp

```diff
@@ -52,6 +52,10 @@ PYBIND11_MODULE(_pycudaq, mod) {
   mod.def("set_random_seed", &cudaq::set_random_seed,
           "Provide the seed for backend quantum kernel simulation.");
 
+  mod.def("get_random_seed", &cudaq::get_random_seed,
```

Comment:
Do you have a use case in mind where a user will need this function? For example, I think Numpy doesn't even provide a function like this, just the set random seed. I'm thinking this should just be some internal runtime function that we can use for `broadcast.h`

Reply:
No, I can't think of any current use cases for Python, so I will remove it.

Reply:
Removed in [e82149d](https://github.com/NVIDIA/cuda-quantum/pull/648/commits/e82149ddd38dc91af191fa8ea1a5b1aa453ad453)